npm/vue/.releaserc.js

```diff
@@ -2,6 +2,6 @@ module.exports = {
   ...require('../../.releaserc.base'),
   branches: [
     '@cypress/vue@1.0.0',
-    { name: 'master', prerelease: 'alpha' },
+    { name: 'master', channel: 'latest', prerelease: 'alpha' },
```

Comment:
Channel latest is the default one, next is the one you want!

```suggestion
    { name: 'master', channel: 'next', prerelease: 'alpha' },
```

Reply:
Hmm... I think I want latest?... There's no stable version.

Reply:
Hmm... I think I want latest?... There's no stable version.